CMakeLists.txt

```diff
@@ -270,47 +272,18 @@ set_target_properties(Fields2Cover
   PUBLIC_HEADER "include/fields2cover.h"
 )
 
-if(ORTOOLS_FETCH_SRC)
-  install(
-    TARGETS
-      Fields2Cover
-      steering_functions
-      matplot
-      ortools
-    EXPORT Fields2Cover-targets
-    LIBRARY DESTINATION ${INSTALL_LIB_DIR}
-    ARCHIVE DESTINATION ${INSTALL_LIB_DIR}
-    RUNTIME DESTINATION ${INSTALL_BIN_DIR}
-    PUBLIC_HEADER DESTINATION ${INSTALL_INCLUDE_DIR}
-  )
-else(ORTOOLS_FETCH_SRC)
-  if(NOT "${ortools_SOURCE_DIR}" STREQUAL "")
-      install(
-        DIRECTORY "${ortools_SOURCE_DIR}/share"
-        DESTINATION "${INSTALL_LIB_DIR}/../"
-      )
-      install(
-        DIRECTORY "${ortools_SOURCE_DIR}/include"
-        DESTINATION "${INSTALL_INCLUDE_DIR}/../"
-      )
-      install(
-        DIRECTORY "${ortools_SOURCE_DIR}/lib"
-        DESTINATION "${INSTALL_LIB_DIR}/../"
-      )
-  endif(NOT "${ortools_SOURCE_DIR}" STREQUAL "")
-
-  install(
-    TARGETS
-      Fields2Cover
-      steering_functions
-      matplot
-    EXPORT Fields2Cover-targets
-    LIBRARY DESTINATION ${INSTALL_LIB_DIR}
-    ARCHIVE DESTINATION ${INSTALL_LIB_DIR}
-    RUNTIME DESTINATION ${INSTALL_BIN_DIR}
-    PUBLIC_HEADER DESTINATION ${INSTALL_INCLUDE_DIR}
+install(
+  TARGETS
+    Fields2Cover
+    steering_functions
+    matplot
+    ${ORTOOLS_TARGET}
```

Comment:
Not sure about this ${ORTOOLS_TARGET} variable. It should only be set with USE_ORTOOLS_FETCH_SRC = ON.  There is probably a better way.

Reply:
If it works, we are fine with it